test/asynchronous/test_auth.py

```diff
@@ -80,6 +82,7 @@ def run(self):
 
 
 class TestGSSAPI(AsyncPyMongoTestCase):
+    pytestmark = pytest.mark.auth
```

Comment:
We can put this once at the top of the file to mark the whole module rather than each class individually.

Reply:
Cool, I didn't think that would work with unittest classes, updated.